src/datachain/model/bbox.py

```diff
@@ -0,0 +1,102 @@
+from pydantic import Field
+
+from datachain.lib.data_model import DataModel
+
+
+class BBox(DataModel):
+    """
+    A data model for representing bounding box.
+
+    Attributes:
+        title (str): The title of the bounding box.
+        coords (list[int]): The coordinates of the bounding box.
+
+    The bounding box is defined by two points:
+        - (x1, y1): The top-left corner of the box.
+        - (x2, y2): The bottom-right corner of the box.
```

Comment:
[Q] We currently support N `File` models on the same record as long as they have different names. Should these models have a file they are associated with?

Reply:
Good question indeed. I am thinking about use cases. Do we have any examples of other signals with a file associated with? 🤔

Reply:
The first one that comes to mind would be extracting images or text or tables from a PDF and having bounding boxes attached to the original for where they've been extracted. 

I'm not criticising or saying we should add this now but this is something to consider as we start adding these model types.

Reply:
> I'm not criticising or saying we should add this now but this is something to consider as we start adding these model types.

I absolutely agree with you! This is very good point I wan't thinking about 👍

> The first one that comes to mind would be extracting images or text or tables from a PDF and having bounding boxes attached to the original for where they've been extracted.

I meant existing implementations or examples of signals for multiple files, was wondering may be you have any from the top of your head.